index.html

```diff
@@ -8,5 +8,8 @@
 <body>
     <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolore ea esse fuga iure molestias odio pariatur provident quis soluta ut!</p>
     <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugit, quod.</p>
+        <div id="background">
+            <p>Dit is de eerste aanpassing in jouw html bestand. Niet bijzonders, gewoon om te oefenen. In css wat aangepast</p>
```

Comment:
Misschien moeten we wel iets meer content toevoegen, user-experience is een beetje mager zo 😆